src/config.js

```diff
@@ -0,0 +1,4 @@
+export const SIGNIN_URL = 'http://10.58.0.54:8000/users/signin';
+export const SIGNUP_URL = 'http://10.58.0.54:8000/users/signup';
+export const NEWREVIEW_URL = 'http://10.58.0.54:8000/reviews/2';
+export const REVIEW_URL = 'http://10.58.0.54:8000/restaurants/2';
```

Comment:
- config.js 에서 관리하시는 것 굿-! 다만, 백엔드 ip 주소 부분을 변수로 처리해주시면 좋을 것 같습니다.
```suggestion
const BASE_URL = 'http://10.58.0.54:8000'
export const SIGNIN_URL = `${BASE_URL}/users/signin`;
export const SIGNUP_URL = `${BASE_URL}/users/signup`;
export const NEWREVIEW_URL = `${BASE_URL}/reviews/2`;
export const REVIEW_URL = `${BASE_URL}/restaurants/2`;
```

Reply:
수정했습니다 !